src/codegen/program.lisp

```diff
@@ -199,7 +198,7 @@ Example:
            (type node-abstraction node)
            (type tc:environment env))
   `(defun ,name ,(node-abstraction-vars node)
-     ,(function-declarations node env)
+     (declare (ignorable ,@(node-abstraction-vars node)))
```

Comment:
We have a Coalton rule where if the name starts with `_` then it is an ignored variable.

Was the old behavior to emit `ignorable` for all variables?

Reply:
From the result of `coalton-codegen`, yes.